### ISSUE TITLE
All product images having links must have alternative text

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13523]
**Describe the bug**

- Some images are missing alternative text, this makes some links fail the  automated test 

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**
1. Go to the search box,  type "Camera Tapes" and trigger search.
2. Scroll down to product CAMERA TAPE DIGITAL8 90MIN 2PK
3. Inspect the product.
4. See error.

**Expected behavior**

- All images contained in a link should have alternative text.

**Screenshots**

![Screen Shot 2021-08-05 at 6 52 28 PM](https://user-images.githubusercontent.com/15113219/130302194-851ca557-6274-4bfb-a487-504e1bf7a897.png)

**Additional context**

- There are exceptions when the alternative text for images is not needed. In that case, there still needs to have the alt attribute, but it should be empty, `alt=""` or the `role="presentation"` to convey to the screen reader that the image has to be ignored/skipped. This doesn't seem to be the case for the search results page.

**Comment by saad-mo** [comment]
@developpeurweb this seems like an issue with missing product sample data. some products do not have a product summary at all which is used in the `alt` attribute. should we have a fallback in place to have some generic alt text when there is nothing coming from the backend data? 

<img width="1294" alt="Screen Shot 2021-10-04 at 3 27 06 PM" src="https://user-images.githubusercontent.com/15215400/135920069-0ac4e5a0-db43-4176-954b-16d6e8e80514.png">

**Comment by developpeurweb** [comment]
@saad-mo yes please, fallback needed, otherwise there's a risk to fail the auto-test every now and then.

**Comment by RadhepS** [comment]
Will be solved by this PR: https://github.com/SAP/spartacus/pull/14088/files

**Comment by BenjaminEmiliani** [comment]
@RadhepS  this ticket can be closed correct?
